File: app/src/main/java/com/mertfatih/bookhabit/model/Book.java
package com.mertfatih.bookhabit.model;

import java.io.Serializable;

public class Book implements Serializable {

    public String book_name;
    public String author_name;
    public String date;
    public int book_page;

    public Book(String book_name, String author_name, String date, int book_page) {
        this.book_name = book_name;
        this.author_name = author_name;
        this.date = date;
        this.book_page = book_page;
    }
}

File: app/src/main/java/com/mertfatih/bookhabit/model/Book.kt
package com.mertfatih.bookhabit.model

import android.os.Parcelable
import androidx.room.*
import kotlinx.android.parcel.Parcelize
import java.time.LocalDateTime
import java.util.*

@Parcelize
@Entity(tableName = "book_table")
data class Book(
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "book_id")
    val id: Int,
    @ColumnInfo(name = "book_name")
    val bookName: String,
    @ColumnInfo(name = "author_name")
    val authorName: String,
    @ColumnInfo(name = "book_page")
    val bookPageCount: Int,

): Parcelable


